dist/js/glightbox.js

```diff
@@ -4,6 +4,20 @@
   (global = global || self, global.GLightbox = factory());
 }(this, (function () { 'use strict';
 
+  function _classCallCheck(a, n) {
```

Comment:
Why did you replace readable variable names with single letter versions? This makes the code harder to read and to maintain